docs/docker_container.md

```diff
@@ -303,7 +303,28 @@ docker run --rm -d -p 9001:9001 \
 -e S3_ENDPOINT=“${S3_ENDPOINT}” \
 openvino/model_server:latest \
 --model_path s3://bucket/model_path --model_name s3_model --port 9001
+```
+
+You can also use anonymous access to s3 public paths.
+
+Example command with `s3://<public_bucket>/<model_path>:`
+
+```
+docker run --rm -d -p 9001:9001 \
+openvino/model_server:latest \
+--model_path s3://public_bucket/model_path --model_name s3_model --port 9001
+```
+
+or setup a profile credentials file in the docker image described here
+[AWS Named profiles](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html)
 
+Example command with `s3://<bucket>/<model_path>:`
+
+```
+docker run --rm -d -p 9001:9001 \
+-e AWS_PROFILE=“${AWS_PROFILE}” \
+openvino/model_server:latest \
+--model_path s3://bucket/model_path --model_name s3_model --port 9001
 ```
```

Comment:
It would be useful to add a general comment to cloud storages, that ovms require "read" and "list" permissions.